pkg/proxy/ipvs/proxier.go

```diff
@@ -362,10 +362,14 @@ func NewProxier(ipt utiliptables.Interface,
 		}
 	}
 
-	// Set the connection reuse mode
-	if val, _ := sysctl.GetSysctl(sysctlConnReuse); val != 0 {
-		if err := sysctl.SetSysctl(sysctlConnReuse, 0); err != nil {
-			return nil, fmt.Errorf("can't set sysctl %s: %v", sysctlConnReuse, err)
+	if kernelVersion.LessThan(version.MustParseGeneric("4.19")) {
```

Comment:
According to https://github.com/kubernetes/kubernetes/pull/82066, the min version was 4.1, was that incorrect?

Reply:
Based on the timestamp here https://kernelnewbies.org/Linux_4.1 and the time when the patch for conn reuse went in https://patchwork.ozlabs.org/patch/444904/ it seems like 4.1 is the min version here

Reply:
yes we really need to set it to 4.1 (otherwise that would be a regression for a lot of users)